src/components/ListOrgs.vue

```diff
@@ -272,6 +276,70 @@ const exportAll = async () => {
   exportCsv(exportData, `roar-${activeOrgType.value}.csv`);
 };
 
+const exportOrgUsers = async (orgType) => {
+  try {
+    // First, count the users
+    const userCount = await countUsersByOrg(activeOrgType.value, orgType.id, orderBy);
+
+    if (userCount === 0) {
+      toast.add({
+        severity: 'error',
+        summary: 'Export Failed',
+        detail: 'No users found for the organization.',
+        life: 3000,
+      });
+      return;
+    }
+
+    if (userCount >= CSV_EXPORT_MAX_RECORD_COUNT) {
+      toast.add({
+        severity: 'error',
+        summary: 'Export Failed',
+        detail: 'Too many users to export. Please filter the users by selecting a smaller org type.',
+        life: 3000,
+      });
+      return;
+    }
+
+    // Fetch the users if the count is within acceptable limits
+    const users = await fetchUsersByOrg(activeOrgType.value, orgType.id, userCount, ref(0), orderBy);
+
+    const computedExportData = users.map((user) => ({
+      Username: _get(user, 'username'),
+      Email: _get(user, 'email'),
+      FirstName: _get(user, 'name.first'),
+      LastName: _get(user, 'name.last'),
+      Grade: _get(user, 'studentData.grade'),
+      Gender: _get(user, 'studentData.gender'),
+      DateOfBirth: _get(user, 'studentData.dob'),
+      UserType: _get(user, 'userType'),
+      ell_status: _get(user, 'studentData.ell_status'),
+      iep_status: _get(user, 'studentData.iep_status'),
+      frl_status: _get(user, 'studentData.frl_status'),
+      race: _get(user, 'studentData.race'),
+      hispanic_ethnicity: _get(user, 'studentData.hispanic_ethnicity'),
+      home_language: _get(user, 'studentData.home_language'),
+    }));
+
+    // ex. cypress-test-district-users-export.csv
+    exportCsv(computedExportData, `${_kebabCase(orgType.name)}-users-export.csv`);
+
+    toast.add({
+      severity: 'success',
+      summary: 'Export Successful',
+      detail: 'Users have been exported successfully!',
+      life: 3000,
+    });
+  } catch (error) {
+    toast.add({
+      severity: 'error',
+      summary: 'Export Failed',
+      detail: error.message,
+      life: 3000,
+    });
+  }
```

Comment:
This is a dangerous one as any error that arises from within the `try` block gets effectively swallowed. Whilst the user gets to see a Toast, we very likely won't have any errors in Sentry about this. 

Again, not holding the PR back for this simply because this isn't the only place where this is happening and I'll write a ticket for it. But, generally speaking, it would be good to manually capture any exception here to ensure Sentry logs it. 

```js
import * as Sentry from '@sentry/vue'

try { 
  // …
} catch {
  toast.add({})
  Sentry.captureException(error)
}
```

Reply:
Ohh this is cool, let me add the sentryCapture